components/http_foundation/trusting_proxies.rst

```diff
@@ -1,55 +0,0 @@
-.. index::
-   single: Request; Trusted Proxies
-
-Trusting Proxies
```

Comment:
this removal is wrong. We still have trusted proxies (even though ESI accepts signed requests from anywhere)